src/localzigzag.jl

```diff
@@ -23,12 +25,14 @@ end
 normsq(x::Real) = abs2(x)
 normsq(x) = dot(x,x)
 
-pos(x) = max(zero(x), x)
+
 function λ(G, ∇ϕ, i, x, θ, Z::LocalZigZag)
     pos(∇ϕ(x, i)*θ[i])
 end
 
-ab(G, i, x, θ, c, Z::LocalZigZag) = c[i]*sqrt(sum(abs2(x[j]) for j in neighbours(G, i))), 1.0
+#ab(G, i, x, θ, c, Z::LocalZigZag) = Z.Γ[:,i]'*(x-Z.μ)*θ[i] + c[i]*sqrt(sum(abs2(x[j]-Z.μ[j]) for j in neighbours(G, i))), 1.0
+ab(G, i, x, θ, c, Z::LocalZigZag) = c[i]*sqrt(sum(abs2(x[j]-Z.μ[j]) for j in neighbours(G, i))), 1.0
```

Comment:
@SebaGraz , can you explain why the slope is b=1? `c[i]*sqrt(sum(abs2(x[j] + t*θ[j] - Z.μ[j])` could also become smaller?

Reply:
If the target is gaussian with mean \mu and precision matrix `\Gamma` then given the initial position and velocity x(0), θ(0) the ith Poisson rate is `\lambda(t) = max(0, θ_i(t)*∇ϕ(x(t)[ i ])) = max(0, θ_i(0)Γ[ i , : ]*x(t)) = max(0, θ_i(0)Γ[ i , : ]*(x(0) + θ(0)*t))` therefore the right `a` and `b` (intersept and slope) should be `a = θ[i]*Γ[ i , : ]*(x(0)` and `b = θ_i(0)Γ[ i , : ]θ[0]`
which in general could be anything but for example if  Γ is the identity then `b  = 1`....maybe this answers your question

Reply:
Yeah, I think I have it, see below